sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -383,13 +384,23 @@ public void deleteAllExpiredFeatureConfigurations(Date date) {
 	public boolean isFeatureDisabled(FeatureType featureType) {
 
 		CriteriaBuilder cb = em.getCriteriaBuilder();
-		CriteriaQuery<Long> cq = cb.createQuery(Long.class);
+		CriteriaQuery<FeatureConfiguration> cq = cb.createQuery(FeatureConfiguration.class);
 		Root<FeatureConfiguration> root = cq.from(FeatureConfiguration.class);
 
-		cq.where(cb.and(cb.equal(root.get(FeatureConfiguration.FEATURE_TYPE), featureType), cb.isFalse(root.get(FeatureConfiguration.ENABLED))));
-		cq.select(cb.count(root));
```

Comment:
you should select only the `FeatureConfiguration.ENABLED` flags

Reply:
changed